boxsdk/client/client.py

```diff
@@ -15,6 +15,8 @@
 from ..object.group_membership import GroupMembership
 from ..util.shared_link import get_shared_link_header
 from ..util.translator import Translator
+from ..pagination.marker_based_object_collection import MarkerBasedObjectCollection
```

Comment:
I don't see any uses of `MarkerBasedObjectCollection` in the diffs? Is this import needed?